components/ad/smartbanner/src/index.js

```diff
@@ -0,0 +1,45 @@
+import React, {Component} from 'react'
```

Comment:
Shouldn't we check the propTypes to be sure all the expected elements are arriving? I mean, with the current code, someone could instantiate the component and forget to add a imageUrl